cmd/mock_server.go

```diff
@@ -48,7 +51,7 @@ func main() {
 	cloudtrace.RegisterTraceServiceServer(grpcServer, &trace.MockTraceServer{})
 	monitoring.RegisterMetricServiceServer(grpcServer, &metric.MockMetricServer{})
 
-	log.Printf("Listening on %s\n", *address)
+	log.Printf("Listening on %s\n", lis.Addr().String())
 
 	if err := grpcServer.Serve(lis); err != nil {
```

Comment:
How do we shutdown the server? Will hitting ctrl+c shutdown gracefully?

It might be good to add an interrupt handler:
```
	c := make(chan os.Signal)
	signal.Notify(c, os.Interrupt, syscall.SIGTERM)
	go func() {
		<-c
		grpcServer.ShutdownGracefully()
	}()
```

Reply:
Yep added that, thanks